xpk.py

```diff
@@ -4795,7 +4804,8 @@ def directory_path_type(value):
     default=1,
     help='The number of nodes to use, default=1.',
 )
-workload_create_parser_optional_arguments.add_argument(
+workload_env_arguments = workload_create_parser_optional_arguments.add_mutually_exclusive_group()
```

Comment:
Do they need to be exclusive?

Reply:
Not necessarily, we can relax this restriction if it would be useful. The idea was just to keep the two flows separate, either you specify the env from a file or manually with `--env`. cc @Obliviour

Reply:
I think from a user perspective, it would be confusing to support both at the same time and handle conflicts between the same variables from both.

LMK if you think otherwise